src/date-picker/date-picker.jsx

```diff
@@ -149,11 +149,14 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: (date) => {
```

Comment:
you shouldn't put it inside the class as this function is treated as static. instead put it outside the class as a top level function. use the impurity of javascript :laughing: